docs/orchestrating-elastic-stack-applications/elasticsearch/orchestration.asciidoc

```diff
@@ -149,7 +149,7 @@ Due to relying on Kubernetes primitives such as StatefulSets, the ECK orchestrat
 ** Single-node clusters
 ** Clusters containing indices with no replicas
 
-If an Elasticsearch node holds the only copy of a shard, this shard becomes unavailable while the node is upgraded. Clusters with more than one node and at least one replica per index are recommended.
+If an {es} node holds the only copy of a shard, this shard becomes unavailable while the node is upgraded. link:{ref}/high-availability-cluster-design.html[Highly available] clusters are recommended, including having more than one node per link:{ref}/data-tiers.html[data tier] and at least one replica per index.
```

Comment:
```suggestion
If an {es} node holds the only copy of a shard, this shard becomes unavailable while the node is upgraded. To ensure link:{ref}/high-availability-cluster-design.html[high availability] it is recommended to configure clusters with three master nodes, more than one node per link:{ref}/data-tiers.html[data tier] and at least one replica per index.
```
Just having multiple nodes per data tier does not make a cluster highly available. ECK starts treating a cluster as HA only if it also has at least three master nodes and this also what the doc says that you propose to link to.